CONTRIBUTOR_LADDER.md

```diff
@@ -1,34 +1,39 @@
 # OSM Contributor Ladder
 
-Hello! We are excited to have you contribute to Open Service Mesh (OSM)! This contributor ladder outlines the different contributor roles within the project, along with the responsibilites and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
+Hello! We are excited to have you contribute to Open Service Mesh (OSM)! This contributor ladder outlines the different contributor roles within the project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up as their involvement in the project grows. Our project members are happy to help you advance along the contributor ladder.
 
-Each of the contributor roles below is organized into lists of three types of things.
-* "Responsibilities" are things that a contributor is expected to do
-* "Requirements" are qualifications a contributor needs to meet to be in that role
-* "Privileges" are things a contributor on that level is entitled to
+Each of the roles is organized into the following categories:
+* "Responsibilities" list what the role is expected to do
+* "Requirements" are qualifications the role needs to meet
+* "Privileges" list what the role is entitled to
 
+| Role | Responsibilities | Requirements | Privileges |
+| -----| ---------------- | ------------ | -------|
+| [Maintainer](#maintainer) | Active member of the community |<ul><li>Contributor for ≥3 months</li><li> Demonstrates deep understanding of multiple areas of the project</li><li>Exercise good judgement for the project</li><li>Maintain contributions to the project</li></ul> | <ul><li>Has GitHub or CI/CD rights to approve any pull requests</li><li>Represent the project as a Maintainer</li><li>Communicate with CNCF on behalf of the project</li></ul> |
+| [Contributor](#contributor) | Following the [CNCF Code of Conduct] and the project [contributing guide] | One or more contributions to the project | Invitations to contributor events |
+| [Community Participant](#community-participant) | Following the [CNCF Code of Conduct] | --- | --- |
```

Comment:
What do you think about reversing this list so the order matches the order in which the roles are presented below? It will also clarify the role progression of community participant to maintainer.